easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.9-gompi-2020b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'SCOTCH'
+version = '6.0.9'
```

Comment:
should be updated to 6.1.0, see https://gforge.inria.fr/forum/forum.php?forum_id=11546...